client/wp-admin-scripts/wc-pay-payments-promotion/wc-payments-row.scss

```diff
@@ -0,0 +1,37 @@
+[data-gateway_id='psuedo_woocommerce_payments'] {
+	> .status a {
+		visibility: hidden !important;
+	}
+
+	> .action {
+		.is-secondary,
+		&.is-secondary:hover {
+			box-shadow: none;
+		}
+
+		a {
+			position: relative;
+			&::after {
+				content: 'Install';
```

Comment:
This might not be as i18n friendly, so I could just hide the previous button instead.
I did this so it doesn't show `Set up` for a quick moment.